app/src/main/java/com/plaid/linksample/network/LinkSampleApi.kt

```diff
@@ -0,0 +1,19 @@
+package com.plaid.linksample.network
+
+import com.google.gson.annotations.SerializedName
+import io.reactivex.Single
+import retrofit2.http.POST
+
+/**
+ * API calls to our localhost token server.
+ */
+interface LinkSampleApi {
+
+  @POST("/get_link_token")
```

Comment:
Replace with "/api/create_link_token" according to https://github.com/plaid/quickstart/commit/bfd0da6bf78c28dfa944e7e31d19891a8dc42737